### DISCUSSION TITLE
Embed name and value on the same line

### DISCUSSION THREAD

**Discussion by frontez** [discord-net/Discord.Net#2233] [Q&A] [answered]
Hi, can you please make an option for embed to have its name and value on the same line like on the picture:

![image](https://user-images.githubusercontent.com/15384653/162277981-dde5eab8-042c-481d-8e2c-0eb11a3a63db.png)

PS this example is not made using Discord.Net

**Comment by CottageDwellingCat** [comment, marked as answer]
That is not possible with Discord embed fields, you could achieve that look with something like
```cs
var inpt = new List<string>()
{
	"cats",
	"are",
	"very",
	"cute",
};

string body = inpt.Select(x => $"**{inpt.IndexOf(x) + 1}:** {x}").Aggregate((x, y) => $"{x}\n{y}");

EmbedBuilder eb = new()
{
	Title = "Cats",
	Description = body,
	Color = Color.Red
};

await channel.SendMessageAsync(embed: eb.Build());
```
![image](https://user-images.githubusercontent.com/80918250/162285697-217b340c-c8d2-4e46-b519-92abf705ed32.png)

**Reply by frontez** [reply]
wow, nice, thank you)
